core/objects.go

```diff
@@ -238,6 +239,16 @@ func (ch Challenge) RecordsSane() bool {
 			return false
 		}
 		return true
+	case ChallengeTypeDNSAccount01:
```

Comment:
Rather than duplicating code in this block, I'd suggest simply changing line 232 to say
```go
	case ChallengeTypeDNS01, ChallengeTypeDNSAccount01:
```